### PR TITLE
Add workflow call event to support reusable workflows

### PR DESCRIPTION
**Pull request by alvintang**
Currently trying out reusable workflows and this PR adds the types to support it.

References:
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_call
https://docs.github.com/en/actions/using-workflows/reusing-workflows

### CODE REVIEW

File: types/events/pull_request_target/types.dhall [hunk 1]
```diff
@@ -1 +1 @@
-< assigned | opened | synchronize | reopened >
+< assigned | opened | synchronize | reopened | closed >
```
Comment (regadas) [inline types/events/pull_request_target/types.dhall:1]: :art: This could be moved to a separate PR

**Review by regadas — COMMENTED** [review]
Per comment on #66 let's wait for a "proper" release so that we can have this one in.

File: types/events/pull_request_target/types.dhall (hunk 1 above)
Comment (alvinBellroy) [reply types/events/pull_request_target/types.dhall:1]: Sorry for the delay! https://github.com/regadas/github-actions-dhall/pull/69

Comment (alvintang) [reply types/events/pull_request_target/types.dhall:1]: sorry for the delay!

**Review by regadas — COMMENTED** [review]
@alvintang sorry, can you rebase this? 🙏

**Review by regadas — APPROVED** [review]
Awesome, thanks!
